chameleon/src/main/java/org/hibernate/omm/dialect/MongodbDialect.java

```diff
@@ -13,6 +13,10 @@
 
 public class MongodbDialect extends Dialect {
 
+	public MongodbDialect() {
```

Comment:
Not sure why but something was looking for this method reflectively

Reply:
This default constructor is used when the other constructor accepting version infos can't be invoked. Guess due to connection reason, the version query interaction failed so the default constructor was invoked.